Assets/Shaders/Goal.shadergraph

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This entire shadergraph seems to be unused and also broken as far as I can tell.

Reply:
I removed this.